src/Panel/IncludePanel.php

```diff
@@ -156,7 +156,7 @@ protected function _isAppFile($file)
      * Check if a path is from a plugin
      *
      * @param string $file File to check
-     * @return bool
+     * @return string|bool plugin name, or false if not plugin
```

Comment:
I dont think an is...() method should ever return anything but bool. Why do you need to return the plugin name?
Then it should be called vendorFile() and return null otherwise.